ads/google/a4a/utils.js

```diff
@@ -132,10 +132,14 @@ export function isReportingEnabled(ampElement) {
  *     Parameters that will be put at the end of the URL, where they may be
  *     elided for length reasons. Intended for parameters with potentially
  *     long values, like URLs.
+ * @param {!Array<string>=} opt_experimentIds Any experiments IDs (in addition
```

Comment:
They're handled as strings everywhere else and it's also the type expected by the mergeExperimentIds function, so would rather just leave as Array